Fast company

Homegor opened the pull request.

Homegor commented:
я забыл промаржить предыдущее задание

Homegor committed:
не могу реализовать поиск через map
bookmarks

Homegor committed:
Merge branch 'main' into Fast-Company

Solexofficial commented on React/fast-company/src/App.js:
```diff
@@ -0,0 +1,35 @@
+    console.log(users)
+    return (
+        <>
+            <SearchStatus length={users.length}/>
```
rename length to usersCount

Homegor replied:
Готово

Solexofficial commented on React/fast-company/src/App.js:
```diff
@@ -0,0 +1,35 @@
+    const handleDelete = (userId) => {
+        setUsers(users.filter(user => user._id !== userId))
+    }
+    console.log(users)
```
а где хендлер переключения избранного?)

Homegor replied:
https://github.com/Homegor/JFD/pull/4/commits/f270c6f8f23ba53dc16184841cdb84c9ec1734c5

Solexofficial commented on React/fast-company/src/components/bookmark.jsx:
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const BookMark = ({ status, ...rest }) => {
+
```
пустой компонент?

Homegor replied:
https://github.com/Homegor/JFD/pull/4/commits/f270c6f8f23ba53dc16184841cdb84c9ec1734c5

Solexofficial commented on React/fast-company/src/components/qualitie.jsx:
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+const Qualitie = ({ color, name, _id }) => {
+    return (
+        <span key={_id} className={`badge m-2 bg-${color}`}>{name}</span>
```
ключи передавай в тот момент когда рендеришь список, в самом мапе, вроде писал про это

Homegor replied:
Исправил в новом

Homegor committed:
Готовое задание bookMark

Homegor committed:
Merge remote-tracking branch 'origin/Fast-Company' into Fast-Company

Homegor committed:
rename length to usersCount

Homegor commented:
Мне даже интересно. У меня 2 pull request, с какими ошибками они объединятся с основной веткой

Homegor committed:
Много правок

Homegor committed:
handleIncrement & handleDecrement
правки prevState и ограничение счетчика значение которого не может быть ниже нуля

Homegor committed:
Merge branch 'main' into Fast-Company
